File: src/main/java/com/bdev/hogwarts_api/data/converter/student_status/StudentStatusDtoConverter.java
package com.bdev.hogwarts_api.data.converter.student_status;

import com.bdev.hogwarts_api.data.dto.student.StudentStatus;
import com.bdev.hogwarts_api.data.model.student_status.StudentStatusModel;

public class StudentStatusDtoConverter {
    public static StudentStatusModel convert(StudentStatus studentStatus) {
        StudentStatusModel studentStatusModel = new StudentStatusModel();

        studentStatusModel.setId(studentStatus.getId());
        studentStatusModel.setStudentId(studentStatus.getStudentId());
        studentStatusModel.setStatus(studentStatus.getStatus());
        studentStatusModel.setCreationTime(studentStatus.getCreationTime());

        return studentStatusModel;
    }
}

File: src/main/java/com/bdev/hogwarts_api/data/converter/student_status/StudentStatusDtoConverter.kt
package com.bdev.hogwarts_api.data.converter.student_status

import com.bdev.hogwarts_api.data.dto.student.StudentStatus
import com.bdev.hogwarts_api.data.model.student_status.StudentStatusModel

object StudentStatusDtoConverter {
    fun convert(studentStatus: StudentStatus): StudentStatusModel {
        val studentStatusModel = StudentStatusModel()

        studentStatusModel.id = studentStatus.id
        studentStatusModel.studentId = studentStatus.studentId
        studentStatusModel.status = studentStatus.status
        studentStatusModel.creationTime = studentStatus.creationTime
        studentStatusModel.actionTime = studentStatus.actionTime

        return studentStatusModel
    }
}
